.pre-commit-config.yaml

```diff
@@ -65,7 +65,7 @@ repos:
   rev: v2.2.5
   hooks:
   - id: codespell
-    args: [-w]
```

Comment:
is this relevant? if not rm?

Reply:
Yes I think this can be removed now that the files are stored externally